core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -8741,6 +8741,48 @@ private void checkLiteral2(String expression, String expected) {
         .dialect(MssqlSqlDialect.DEFAULT).ok(mssqlExpected);
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-6655">[CALCITE-6655]
+   * Aggregation of deeply nested aggregations </a>.
```

Comment:
extra space before closing tag
You need to use the exact same message as in the JIRA case.
Please use this for your PR as well.
These conventions help us keep track of the correspondence between commits, PRs, and JIRA cases.